src/rest_framework_jwt/utils.py

```diff
@@ -68,6 +69,7 @@ def jwt_create_payload(user):
     expiration_time = issued_at_time + api_settings.JWT_EXPIRATION_DELTA
 
     payload = {
+        'jti': uuid.uuid4(),
```

Comment:
I remember you saying that this could be a separate handler. I think the current implementation is fine and there is no need for enabling customizations at this point.

Reply:
I prefer not making it customizable. Thanks.